variables.tf

```diff
@@ -29,6 +29,12 @@ variable "service_healthcheck_url" {
   EOF
 }
 
+variable "alert_health_check_duration" {
+  type        = number
+  default     = 300
+  description = "How long the synthetics monitor check threshold must fail before an alert is raised (in seconds)"
```

Comment:
For consistency with other comments in this PR, let's use triggered instead of raised.

Reply:
Also, it's the monitor check that fails, not the threshold. So, I'd remove "threshold" from the sentence.